src/pages/UnlockedWallet/OverviewPage/AmountsOverviewPanel.tsx

```diff
@@ -26,89 +26,116 @@ import styled from 'styled-components'
 import Amount from '@/components/Amount'
 import Button from '@/components/Button'
 import { useAppDispatch, useAppSelector } from '@/hooks/redux'
+import AddressOptionsModal from '@/modals/AddressOptionsModal'
 import ModalPortal from '@/modals/ModalPortal'
+import ReceiveModal from '@/modals/ReceiveModal'
 import SendModalTransfer from '@/modals/SendModals/SendModalTransfer'
 import SettingsModal from '@/modals/SettingsModal'
 import { walletLocked } from '@/storage/app-state/slices/activeWalletSlice'
-import { selectAllAddresses } from '@/storage/app-state/slices/addressesSlice'
+import { selectAddressByHash, selectAllAddresses } from '@/storage/app-state/slices/addressesSlice'
 import { useGetPriceQuery } from '@/storage/app-state/slices/priceApiSlice'
 import { getAvailableBalance } from '@/utils/addresses'
 import { currencies } from '@/utils/currencies'
 
 interface AmountsOverviewPanelProps {
   isLoading?: boolean
   className?: string
+  addressHash?: string
 }
 
-const AmountsOverviewPanel = ({ className, isLoading }: AmountsOverviewPanelProps) => {
+const AmountsOverviewPanel = ({ className, isLoading, addressHash }: AmountsOverviewPanelProps) => {
```

Comment:
FYI: This will probably be redesigned a bit. But don't worry, this is only be a cosmetic evolution!